microcosm_postgres/encryption/api.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import event
+
+from microcosm_postgres.encryption.encryptor import Encryptor
+
+
+def init_encryption(encryptor: Encryptor, model_class, key: str) -> None:
+
+    @event.listens_for(model_class, "init")
+    def receive_init(target, args, kwargs):
+        """
+        Encrypt (and delete) plaintext on init.
```

Comment:
`init` is specifically initializes an instance of the model, not when an instance of the model is created from the db (which is load), right? I think I saw that nuance in the documentation and see it reflected here, but for someone who won't dig in might be worth emphasizing.

Reply:
Agreed. I am not content with this naming or even these mechanics for initialization.